examples/basic_operations/add_responsive_search_ad.rb

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env ruby
+# Encoding: utf-8
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This example adds a Gmail ad to a given ad group. The ad group's campaign
+# needs to have an AdvertisingChannelType of DISPLAY and
+# AdvertisingChannelSubType of DISPLAY_GMAIL_AD. To get ad groups, run
+# get_ad_groups.rb.
+
+require 'optparse'
+require 'date'
+require 'open-uri'
+require 'google/ads/google_ads'
+require 'google/ads/google_ads/v1/errors/errors_pb'
```

Comment:
Is this required? We'd rather have examples be version agnostic and rely on "latest" so we don't have to manually update them all the time.